src/reynir/Greynir.grammar

```diff
@@ -4323,16 +4323,32 @@ NlStak_p3_ft_nf_hk → Manns_hk_nf
 NlStak_p3_ft_þf_hk → Manns_hk_þf
 NlStak_p3_ft_þgf_hk → Manns_hk_þgf
 
+NlStak_p3_et_nf_kk → VillaManns_nf
```

Comment:
The right place for these rules might be in the `#if(include_errors)` section at the end of Greynir.grammar.

Reply:
Done; added a description of each error rule chapter